core/src/main/java/org/stellar/anchor/config/Sep10Config.java

```diff
@@ -3,23 +3,75 @@
 import java.util.List;
 
 public interface Sep10Config {
+  /**
+   * The `web_auth_domain` property of <a
+   * href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#response">SEP-10</a>
+   * If the `web_auth_domain` is not specified, the `web_auth_domain` will be set to the domain of
+   * the value of the `home_domain`. `web_auth_domain` value must be equal to the host of the SEP
+   * server.
+   *
+   * @return the `web_auth_domain` property of SEP-10
+   */
   String getWebAuthDomain();
 
+  /**
+   * The `home_domain` property of <a
+   * href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#request">SEP-10</a>
+   * `home_domain` value must be equal to the host of the toml file. If sep1 is enabled, toml file
+   * will be hosted on the SEP server.
+   *
+   * @return the `home_domain` property of SEP-10
+   */
   String getHomeDomain();
 
+  /**
+   * Set if the client attribution is required. Client Attribution requires clients to verify their
+   * identity by passing a domain in the challenge transaction request and signing the challenge
+   * with the ``SIGNING_KEY`` on that domain's SEP-1 stellar.toml. See the SEP-10 section `Verifying
+   * Client Application Identity` for more information, refer to <a
+   * href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#verifying-client-application-identity">SEP-10</a>.
+   *
+   * @return true if the client attribution is required
+   */
   boolean isClientAttributionRequired();
 
+  /**
+   * Whether to enable SEP-10 for this instance of AP server.
+   *
+   * @return true if SEP-10 is enabled
+   */
   Boolean getEnabled();
 
+  /**
+   * Set the authentication challenge transaction timeout in seconds. An expired signed transaction
+   * will be rejected. This is the timeout period the client must finish the authentication process.
+   * (ie: sign and respond the challenge transaction).
+   *
+   * @return the authentication challenge transaction timeout in seconds
+   */
   Integer getAuthTimeout();
 
+  /**
+   * The timeout in seconds of the authenticated JSON Web Token. An expired JWT will be rejected.
+   * This is the timeout period after the client has authenticated.
+   *
+   * @return the timeout in seconds of the authenticated JSON Web Token
+   */
   Integer getJwtTimeout();
 
-  List<String> getClientAttributionDenyList();
+  boolean isRequireKnownOmnibusAccount();
```

Comment:
Don't we need to change these methods to match the config?

Reply:
Ah... missed it in PR merge. It is now fixed.